docs/src/examples/components/Embed/Types/EmbedExampleIframe.shorthand.tsx

```diff
@@ -10,6 +10,7 @@ const EmbedExampleIframe = () => (
       height: 450,
       width: 600,
     }}
+    control={{ name: 'search' }}
```

Comment:
```suggestion
    control='search'
```
nit: can be just icon's name